frontend/components/Workspaces/WorkspaceDirectoryItem.tsx

```diff
@@ -16,27 +16,27 @@ const StyledContainer = styled.div`
 `;
 
 const StyledLink = styled.a`
-  ${({ theme }) => `
   display: flex;
   text-decoration: none;
 
+  &:hover {
+    text-decoration: underline;
+    cursor: pointer;
+  }
+
+  &:active {
+    ${({ theme }) => `
+        color: ${theme.colorNhsukBlack};
+        background-color: ${theme.colorNhsukYellow};
+      `}
```

Comment:
You can bring the `theme` even closer to the usage. And then Prettier is even able to format the CSS for us.
```suggestion
        color: ${({ theme }) => theme.colorNhsukBlack};
        background-color: ${({ theme }) => theme.colorNhsukYellow};
```